geomstats/geometry/dirichlet_distributions.py

```diff
@@ -235,6 +239,78 @@ def coefficients(ind_k):
 
         return gs.squeeze(christoffels)
 
+    def jac_christoffels(self, base_point):
+        """Compute the Jacobian of the Christoffel symbols.
+
+        Compute the Jacobian of the Christoffel symbols of the
+        Fisher information metric.
```

Comment:
Would you have a reference that explains the computations performed in this function? See e.g. here to add a reference to a docstring:
https://github.com/geomstats/geomstats/blob/d895d33c83ba44229c6f8b9b6deebe457b1b5964/geomstats/geometry/stiefel.py#L211

Reply:
It would be great if it could also explain/define the notations `g_y, g_t, df_y, df_t`, that are hard to understand just by reading the code (at least for me!). Alternatively, the variable names `g_y` etc could be replaced by names that are more self-explanatory. Thanks!

Reply:
The notations ```f_y, f_t, g_y, g_t``` etc. simply referred to intermediary terms in the computations. I have renamed them in a more neutral way.

Reply:
I haven't put these computations in any publication yet. But I have added a reference for the computation of the Christoffel symbols.

Reply:
Ok perfect, thank you! 🙏